### DISCUSSION TITLE
How can I build and push artifacts for self managed releases?

### DISCUSSION THREAD

**Discussion by weeco** [goreleaser/goreleaser#1927] [Q&A] [answered]
Hey,
I prefer creating the GitHub releases on my own instead of having goreleaser create them automatically upon new tags. However I still want it to build my go binaries and push it into the respective release every time a new release is created.

I couldn't find a way to achieve this. Is this supported?

**Comment by caarlos0** [comment, marked as answer]
you mean the changelog?

If so, check `goreleaser release --help`.

**Reply by weeco** [reply]
@caarlos0 I don't have the cli installed locally and therefore I'm looking at https://goreleaser.com/customization/release/ . This page indicates that the release feature is in charge of **creating** the release. However I don't want goreleaser to create the release, I just want it to create the artifacts and upload them for the release with the corresponding tag.

I think the only option currently is that goreleaser creates the release with no notes attached at all (which I don't like because of the RSS feeds which would be empty upon new releases)?

**Reply by caarlos0** [reply]
not sure if that's what you want, but if you create the release beforehand (either manually or with other tools), goreleaser will just upload the artifacts and will not touch the release notes at all...

**Reply by caarlos0** [reply]
will mark as resolved due to lack of response, feel free to reply otherwise..
